.changeset/loud-lamps-visit.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': minor
```

Comment:
Should be patch, no? 😅

Reply:
Done in https://github.com/spotify/backstage/pull/2787/commits/2896a6f02d8033b84780612e6513598320caa4fc